Mithril update 2024-01-17 (#323)
IntersectMBO/cardano-updates: blog/2024-01-17-mithril.md

---
title: Mithril Team Update
slug: 2024-01-17-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team completed the implementation of the client in the explorer to enable direct verification of certificates from the browser. They started working on the implementation of a new type of entity to sign for certifying the Cardano transactions set in Mithril networks. They also completed the upgrade of the devnet to support the Conway era, and they activated the Mithril era marker reader on the Cardano chain in the end to end test. Additionally, the team kept implementing a mock aggregator to strengthen Mithril client tests in WASM, and continued working on threat modeling and risk analysis for P2P networking.

Reviewer: ```suggestion
This week, the Mithril team completed the implementation of the client in the explorer, enabling direct certificate verification from the browser. They initiated the implementation of a new entity type for signing to certify the Cardano transactions set in Mithril networks. The team also upgraded the devnet to support the Conway era and activated the Mithril era marker reader on the Cardano chain in the end-to-end test. Additionally, they kept implementing a mock aggregator to strengthen Mithril client tests in WASM and continued working on threat modeling and risk analysis for P2P networking.
```

> Finally, they worked on enhancing node communications between Mithril and Cardano, and they started rolling out the Cardano node in the infrastructure to version `8.7.3`.

Reviewer: ```suggestion
Finally, they worked on enhancing node communications between Mithril and Cardano and started upgrading the infrastructure to support node version `8.7.3`.
```

## Low level overview

> - Completed the issue **Light Wallet: Create new signed entity type for `CardanoTransactions`** [#1434](https://github.com/input-output-hk/mithril/issues/1434)

Reviewer: ```suggestion
- Completed the issue **Light wallet: create a new signed entity type for `CardanoTransactions`** [#1434](https://github.com/input-output-hk/mithril/issues/1434)
```

> - Completed the issue **Implement the mithril client WASM package in the explorer** [#1409](https://github.com/input-output-hk/mithril/issues/1409)

Reviewer: ```suggestion
- Completed the issue **Implement the Mithril client WASM package in the explorer** [#1409](https://github.com/input-output-hk/mithril/issues/1409)
```

- Completed the issue **Activate `babbage`/`conway` eras on `devnet`** [#1425](https://github.com/input-output-hk/mithril/issues/1425)

> - Completed the issue **Transition Mithril Era Reader adapter to `CardanoChain` in `devnet`** [#1428](https://github.com/input-output-hk/mithril/issues/1428)

Reviewer: ```suggestion
- Completed the issue **Transition Mithril era reader adapter to `CardanoChain` in `devnet`** [#1428](https://github.com/input-output-hk/mithril/issues/1428)
```
